From 8f0ef9a7b9af190e49ee74adc6232f43374012e3 Mon Sep 17 00:00:00 2001
From: Luca Colagrande <luca.colagrande3@gmail.com>
Date: Wed, 19 Apr 2023 14:05:28 +0200
Subject: [PATCH] Allow hash comments in assembly

---
 licence-checker/licence-checker.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/licence-checker/licence-checker.py b/licence-checker/licence-checker.py
index e863f94..6263f50 100755
--- a/licence-checker/licence-checker.py
+++ b/licence-checker/licence-checker.py
@@ -142,8 +142,8 @@ COMMENT_CHARS = [
     # Software Files
     ([".c", ".h", ".inc", ".cc", ".cpp"], SLASH_SLASH),  # C, C++
     ([".def"], SLASH_SLASH),  # C, C++ X-Include List Declaration Files
-    ([".S"], [SLASH_SLASH, SLASH_STAR]),  # Assembly (With Preprocessing)
-    ([".s"], SLASH_STAR),  # Assembly (Without Preprocessing)
+    ([".S"], [SLASH_SLASH, SLASH_STAR, HASH]),  # Assembly (With Preprocessing)
+    ([".s"], [SLASH_STAR, HASH]),  # Assembly (Without Preprocessing)
     ([".ld"], SLASH_STAR),  # Linker Scripts
     ([".rs"], SLASH_SLASH),  # Rust
     ([".go"], SLASH_SLASH),  # Golang
-- 
2.28.0

